src/components/Home.jsx

```diff
@@ -14,40 +15,60 @@ export default function Home() {
     genre: 'Jazz',
     instrument: 'Piano',
   });
-  const handleFilterGenre = () => {
-
+  const handleFilterGenre = (e) => {
+    apiMasters.searchHomePageGenre(e.target.innerHTML)
+      .then((result) => setArtists(result.data));
```

Comment:
I could be wrong, but I believe this could be e.target.value and get the same outcome.